bindings/javascript/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "dyne-did",
```

Comment:
Could you please rename this to `@dyne/did` so that npm packages goes into the `@dyne` umbrella ;p